### ISSUE TITLE
Add Tooltip to VerifiedIcon to explain meaning

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/front-end-monorepo#579]
**Package**

app-project

**Feature or Issue Description**

The current approved icon has a title / aria-label, but it'd probably be more helpful to have a tooltip to explain what it actually means, and possibly to a link for more information. Grommet doesn't have a tooltip component, so we'd have to build / find one.

cc #578

**Comment by srallen** [comment]
The `Drop` and `DropButton` is intended for use in tool tip cases. I built a one with `Drop` in the components repo, but it may still need some work.

**Comment by rogerhutchings** [comment]
Example tooltip: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js
